hail/python/hailtop/batch/job.py

```diff
@@ -343,6 +344,48 @@ def always_run(self, always_run: bool = True) -> 'Job':
         self._always_run = always_run
         return self
 
+    def regions(self, regions: Optional[Union[str, List[str]]]) -> 'Job':
+        """
+        Set the cloud regions a job can run in.
+
+        Notes
+        -----
+        Can only be used with the :class:`.backend.ServiceBackend`.
```

Comment:
The job will still execute with the LocalBackend but the regions argument will have no effect, right?

Reply:
That's not what we have been doing with `always_run` etc. I think we should keep the same behavior in case we have other backends in the future where the behavior of ignoring the parameter might give a false sense of security that we're taking regions into account.